Add example for URLs via Liquid (#3703)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/email/templates/link_aliasing.md

{: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3}
{%endraw%}

## Link aliasing for URLs generated via Liquid

For URLs that are generated by an `assign` statement in the HTML or in a Content Block, we recommend adding a question mark (?) into the anchor tag. This will help Braze append query parameters (`lid = somevalue`) so link aliasing can work properly. Without identifying where to append query parameters, link aliasing will not recognize these URLs.

### Example

> Check out this example using the `assign` statement.

Reviewer: ```suggestion
Check out the following link aliasing example for the recommended formatting of the anchor tag:
```

{% raw %}

```liquid
{% assign link1 = https://www.braze1.com %}

> {% assign link2 = https://www.braze2.com %}

Reviewer: ```suggestion
{% assign link2 = https://www.braze2.com %}

<a href="{{link2}}?">{{link2}}?</a>
```

```

> As another example, `<a href="{{link1}}">{{link1}}</a>` will not work whereas `<a href="{{link2}}?">{{link2}}?</a>` works since there's a question mark in the anchor tag.

Reviewer: ```suggestion
```
